swap/src/monero.rs

```diff
@@ -1,143 +1,377 @@
+pub mod wallet;
+
+use ::bitcoin::hashes::core::fmt::Formatter;
```

Comment:
```suggestion
```

This like a mistake where CLion imports `fmt` from `rust_bitcoin` instead of `std`